api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -125,4 +127,12 @@ public CohortDefinition getDiedDuringPeriod() {
         cd.addParameter(new Parameter("diedOnOrBefore", "reporting.parameter.endDate", Date.class));
         return new MappedParametersCohortDefinition(cd, "diedOnOrAfter", "startDate", "diedOnOrBefore", "endDate");
     }
+
+    @DocumentedDefinition("personWithAttribute")
+    public CohortDefinition getPersonWithAttribute() {
+        PersonAttributeCohortDefinition cd = new PersonAttributeCohortDefinition();
+        cd.addParameter(new Parameter("attributeType", "reporting.parameter.attributeType", PersonAttributeType.class));
+        cd.addParameter(new Parameter("values", "reporting.parameter.values", String.class, List.class, null));
+        return new MappedParametersCohortDefinition(cd, "attributeType", "attributeType", "values", "values");
```

Comment:
I don't think this line is adding any value.  Can't you just return "cd" if you aren't changing the names or mappings of any of the parameters?